dotcom-rendering/src/web/layouts/Liveblog.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect } from 'react';
+
+import { breakpoints } from '@guardian/source-foundations';
+
+import { decideFormat } from '../lib/decideFormat';
+
+import { Live } from '../../../fixtures/generated/articles/Live';
+
+import { embedIframe } from '../browser/embedIframe/embedIframe';
+import { mockRESTCalls } from '../lib/mockRESTCalls';
+import { injectPrivacySettingsLink } from '../lib/injectPrivacySettingsLink';
+
+import { extractNAV } from '../../model/extract-nav';
+import { DecideLayout } from './DecideLayout';
+import { doStorybookHydration } from '../browser/islands/doStorybookHydration';
+
+mockRESTCalls();
+
+export default {
+	title: 'FullPage/LiveBlog',
+	parameters: {
+		chromatic: {
+			viewports: [breakpoints.wide],
+			pauseAnimationAtEnd: true,
+		},
+	},
+};
+
+// HydratedLayout is used here to simulated the hydration that happens after we init react on
+// the client. We need a separate component so that we can make use of useEffect to ensure
+// the hydrate step only runs once the dom has been rendered.
+const HydratedLayout = ({
```

Comment:
```suggestion
/**
 * HydratedLayout is used here to simulated the hydration that happens after we init react on
 * the client. We need a separate component so that we can make use of `useEffect` to ensure
 * the hydrate step only runs once the DOM has been rendered.
 */
const HydratedLayout = ({
```